benchmarks/microbenchmarks/operatorbench.py

```diff
@@ -179,6 +179,7 @@ def benchmark(suite, op, dtype, max_samples, accuracy_checking):
             except Exception as e:
                 print(f"error {operator}")
                 print(e)
+                raise e
```

Comment:
if this is intentional remove the `pass` on the next line

Reply:
done